client/app/main/pdf-upload/pdf-upload.controller.js

```diff
@@ -51,21 +51,24 @@ function PDFUploadCtrl($scope, PDF, AbsenceRecord, Auth, School, toastr) {
     }, $scope.parsedRecord, function(res) {
       resetState();
       var schoolName = res.record.school.name;
-      if (res.students.length) {
+      if (res.outreaches.length) {
+        toastr.info(
+          res.outreaches.length + ' outreaches triggered.',
+          schoolName,
+          {timeOut: 5000}
+        );
```

Comment:
I noticed that some of our toastrs have a dismiss feature and others don't. Is there a common format we want to follow in terms of timeout and dismissible, or is it just a case by case basis?